air-quality-ui/src/components/single-city/SingleCity.module.css

```diff
@@ -34,6 +34,7 @@
     align-content: center;
     width: 600px;
     height: 400px;
+    text-align: center;
```

Comment:
what does this do?

Reply:
Makes the loading spinners align to the middle of the div when loading the colours/lines takes a bit longer. Previously they were left aligned which looked a bit weird

Reply:
Actually it did something bizarre to the text I've just noticed so going to revert and try something else